xlwings/_xlwindows.py

```diff
@@ -135,6 +135,8 @@ def count_columns(xl_sheet, range_address):
 
 
 def get_range_from_indices(xl_sheet, first_row, first_column, last_row, last_column):
+    if 0 in (first_row, first_column, last_row, last_column):
```

Comment:
This check should be done in `main.py` as otherwise you need to duplicate things for Mac.